.azure-devops/create-release.yml

```diff
@@ -6,7 +6,7 @@ trigger: none
 parameters:
 - name: variableGroup
   type: string
-  default: 'aziotcli_test_primary'
+  default: 'vilit-test-variables'
```

Comment:
Placeholder to revert this back when you are done with testing.

Reply:
Also the parameter is meant to be overridable, you don't need a code change to use a different value. Then re-running ( "Run New") from a prior run retains parameter value changes.

Reply:
done